tests/unit/stream_alert_rule_processor/test_sink.py

```diff
@@ -13,28 +13,35 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+# pylint: disable=protected-access
 from datetime import datetime
 import os
+import unittest
 
 from botocore.exceptions import ClientError
-from mock import patch
+from mock import ANY, call, patch
 from nose.tools import assert_equal
 
 from stream_alert.rule_processor.config import load_env
 from stream_alert.rule_processor.sink import StreamSink
 from tests.unit.stream_alert_rule_processor.test_helpers import get_mock_context
 
 
-class TestStreamSink(object):
+@patch.dict(os.environ, {'CLUSTER': 'corp'})
+class TestStreamSink(unittest.TestCase):
```

Comment:
What's the benefit of making this a subclass of `unittest.TestCase` here?

Reply:
You can leverage the built-in test utilities. For example, setting `self.maxDiff = None` will show all diffs (not just snippets) during equality checks. In fact, I think we can use the built in `self.assertEqual()` instead of using the one from `nose`, but still be able to run the tests with nose, but that's a project for another day.

It looks like we only inherit from `unittest.TestCase` in the tests I created :P So I'll remove them to be consistent